package/create_source_tarball.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+#
+# Creates source tarballs for SuperCollider in the form of
+# 'SuperCollider-x.x.x-Source.tar.bz2' and moves the file to the location of
+# this script (the repository's package folder).
+#
+# Requirements:
+# - git
+# - a writable (user) /tmp folder for mktemp
+# - gnupg >= 2.0.0 (if source tarball signing is requested)
+# - a valid PGP key in the keyring (if source tarball signing is requested)
+
+set -euo pipefail
+
+get_absolute_path() {
+  cd "$(dirname "$1")" && pwd -P
+}
+
+validate_project_tag() {
+  if ! git ls-remote -t "${upstream}"| grep -e "${version}$" > /dev/null; then
+    echo "The tag '$version' could not be found in upstream repository (${upstream})."
+    exit 1
+  fi
+}
+
+checkout_project() {
+  echo "Cloning project below working directory ${working_dir}"
+  cd "$working_dir"
+  git clone "$upstream" --branch "$version" \
+                        --single-branch \
+                        --recursive \
```

Comment:
`--recurse-submodules` is preferred, i think `--recursive` may even be deprecated. also strongly recommend `--depth=1`, this speeds things up massively

Reply:
Ah yes. I wasn't too sure about the depth parameter, but this will probably be very beneficial.

Woah, didn't know that `--recursive` is potentially deprecated. It works (and doesn't throw a warning either). :O